src/SearchFilters.js

```diff
@@ -77,13 +85,16 @@ let BuilderSearchFilters = ({ setMode, trees, BuilderFilters }) => (
 let SearchFilters = ({
   mode,
   setMode,
+  disableAdvanced,
```

Comment:
I think it's ok if we add the extra word here for clarity and just call it `disableAdvancedSearch`

Reply:
Yeah good point, `disableAdvancedMode` to be consistent with other props

Reply:
Yeah that sounds good 👍